.github/workflows/CI.yml

```diff
@@ -1164,7 +1164,7 @@ jobs:
         run: |
           opts=""
           if [[ -f testing-framework/terraform/testcases/${{ matrix.testcase }}/parameters.tfvars ]] ; then opts="-var-file=../testcases/${{ matrix.testcase }}/parameters.tfvars" ; fi
-          cd testing-framework/terraform/eks && terraform init && terraform apply -auto-approve $opts -var="aoc_version=${{ needs.e2etest-preparation.outputs.version }}" -var="testcase=../testcases/${{ matrix.testcase }}"
+          cd testing-framework/terraform/eks && terraform init && terraform apply -auto-approve $opts -var="aoc_version=${{ needs.e2etest-preparation.outputs.version }}" -var="aoc_image_repo=public.ecr.aws/${{ env.ECR_REPO }}" -var="testcase=../testcases/${{ matrix.testcase }}"
```

Comment:
The `aoc_image_repo` is being taken from [here](https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/common.tf#L33), we probably don't need this variable. since the tests are already using the [expected image](https://github.com/aws-observability/aws-otel-collector/runs/5795966122?check_suite_focus=true#step:8:122).

Reply:
Good catch, will close this